src/osd/SnapMapper.cc

```diff
@@ -119,7 +119,8 @@ int OSDriver::get_next(
     ch, hoid, key
   ).safe_then_unpack([&key, next] (bool, FuturizedStore::Shard::omap_values_t&& vals) {
     CRIMSON_DEBUG("OSDriver::get_next key {} got omap values", key);
-    if (auto nit = std::begin(vals); nit == std::end(vals)) {
+    if (auto nit = std::begin(vals);
+        nit == std::end(vals) || !SnapMapper::is_mapping(nit->first)) {
```

Comment:
we need return SnapMapper entries or not SnapMapper entries? !SnapMapper::is_mapping(nit->first) if I understand right, not SnapMapper will return error code -ENOENT.

Reply:
We use `omap_get_values` here, we expect to get Snapmapper mappings (SNA_) entries.
The issue is that if no mappings exist `omap_get_values` will return other keys (_biginfo, _epoch ..) - 
we should ignore them.